packages/neptune-extension/src/common/components/checkout-modal/CheckoutModal.tsx

```diff
@@ -55,7 +55,7 @@ const CheckoutModal: React.FC<CheckoutModalProps> = ({
 
   const [ projectId, projectLabel, projectInputProps, projectMetaProps ] = useSelectInputValue(
     initialProjectId,
-    () => fetchProjectOptions('readable'),
+    () => fetchProjectOptions('writable'),
```

Comment:
Why?
I explicitly changed it to readable.
I don't see any reason to prevent user to checkout notebook from any readable project.